languages/example/hostserv.lang.json

```diff
@@ -18,4 +18,5 @@
   "Syntax: $bSET <user> <vhost>$b\n\nSET sets a user's vhost, bypassing the request system.": "Syntax: $bSET <user> <vhost>$b\n\nSET sets a user's vhost, bypassing the request system.",
   "Syntax: $bSTATUS [user]$b\n\nSTATUS displays your current vhost, if any, and the status of your most recent\nrequest for a new one. A server operator can view someone else's status.": "Syntax: $bSTATUS [user]$b\n\nSTATUS displays your current vhost, if any, and the status of your most recent\nrequest for a new one. A server operator can view someone else's status.",
   "Syntax: $bWAITING$b\n\nWAITING shows a list of pending vhost requests, which can then be approved\nor rejected.": "Syntax: $bWAITING$b\n\nWAITING shows a list of pending vhost requests, which can then be approved\nor rejected.",
+  "^[0-9A-Za-z.\\-_/]+$": "^[0-9A-Za-z.\\-_/]+$"
```

Comment:
this shouldn't be in the translation file, think I used to remove this string manually

Reply:
oh yeah, thanks